hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -439,6 +460,14 @@ msgstr "wird an der Kasse angewendet"
 msgid "Billing Address"
 msgstr "Rechnungsadresse"
 
+msgid ""
+"Please select one of the cards that you used before or fill in your credit "
+"card information below. We are using <a href=\"https://stripe.com\" target="
+"\"_blank\">Stripe</a> for payment and do not store your information in our "
+"database."
+msgstr ""
```

Comment:
@MalcolmA for the above string ?

Reply:
I think I already did this, maybe at the wrong place.

+msgid ""
+"Please select one of the cards that you used before or fill in your credit "
+"card information below. We are using <a href=\"https://stripe.com\" target="
+"\"_blank\">Stripe</a> for payment and do not store your information in our "
+"database."
+msgstr 

Bitte wähle eine der zuvor genutzten Kreditkarten oder gib Deine Kreditkartendetails unten an. Die Bezahlung wird über <a href="https://stripe.com\" target=""_blank">Stripe abgewickelt. Wir speichern Deine Kreditkartendetails nicht in unserer Datenbank.

Reply:
You are right. We have some code duplication for landing/hosting pages.